scripts/envsetup.sh

```diff
@@ -3,9 +3,13 @@
 SCRIPT="envsetup.sh"
 MACHINE="$1"
 BUILDDIR="build"
+DISTRO="poky-sota-systemd"
+BASE_CONF="local.conf.base.append"
+declare -A supported_distros=( ["poky-sota-systemd"]="local.conf.systemd.append" ["poky-sota"]="local.conf.base.append" )
 
-[[ "$#" -lt 1 ]] && { echo "Usage: ${SCRIPT} <machine> [builddir]"; return 1; }
+[[ "$#" -lt 1 ]] && { echo "Usage: ${SCRIPT} <machine> [builddir] [distro=< poky-sota-systemd | poky-sota >]"; return 1; }
 [[ "$#" -eq 2 ]] && { BUILDDIR="$2"; }
```

Comment:
This `-eq` should actually be `-ge`. Currently if three args are provided, the build dir arg is ignored.

Reply:
corrected it.